tests/integration_test.py

```diff
@@ -1073,6 +1082,64 @@ def runTest(self):
         self.assertTrue(sock.fileno() > -1)
 
 
+class TestRunContainerReadingSocket(BaseTestCase):
```

Comment:
I have no issue parsing `nxt` into `next` personally, so that doesn't bother me. 

@mnowster Is there a specific issue with those? Considering we're operating in python where `len` and `hasattr` and `def` are common keywords, I believe there's leeway for at least some abbreviation that still preserves clarity and readability.